src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -959,6 +983,24 @@ SpinelNCPInstance::set_property(
 				.finish()
 			);
 
+		} else if (strcaseequal(key.c_str(), kWPANTUNDProperty_BorderAgentProxyStream)) {
+			Data packet = any_to_data(value);
+
+			uint16_t port = *(uint16_t*)(packet.data() + packet.size() - sizeof(port));
```

Comment:
Yes, this is encoded in network byte order, as you see in the next line, it converts it into host byte border. I will update the implementation here to make it more clear.